DEVELOPER_GUIDE.md

```
@@ -0,0 +1,530 @@
+
+### Option 1: Dev Container (Recommended)
+
+1. Install [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+2. Open the repository in VS Code
+3. When prompted, reopen the project in the container
+4. All dependencies and tools are pre-configured
+
+### Option 2: Local Development
+
+1. **Clone the repository:**
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)